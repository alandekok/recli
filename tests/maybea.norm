([a] smile|have [a] good day|hello [a])
