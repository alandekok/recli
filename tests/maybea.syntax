hello [a]
# Allowed, but not really a good idea.
[a] smile
have [a] good day
