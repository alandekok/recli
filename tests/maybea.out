hello
hello a
a smile
smile
have a good day
have good day
